config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml

```diff
@@ -12,7 +12,6 @@ spec:
   services:
     - configure-network
     - validate-network
-    - download-cache
```

Comment:
why is not listed before configure-network?

Reply:
it wasn't listed at all, probably i forgot to add it in the original PR and i think that other sample files could suffer of the same issue. I'll propose a followup PR to fix it after this one is get merged. WDYT?

Reply:
Yes, please do a follow up. :+1: